### DISCUSSION TITLE
Does the new 4.89 update include and of the suggestions or bug fixes I reported?

### DISCUSSION THREAD

**Discussion by ghost** [aldostools/IRISMAN#57] [Q&A] [answered]
Title… I’m just curious to know before updating from the 4.88 version.

**Comment by aldostools** [comment, marked as answer]
No. Thw latest build of 4.89 only includes an update to polish language and the update for mamba & showtime.

I haven't had time to work on IRISMAN. If you submit a pull request with the changes would be easier for me to compile a fixed version.
